moped-editor/src/views/projects/projectView/ActivityLogComponents/ProjectActivityEntry.js

```diff
@@ -0,0 +1,149 @@
+import React from "react";
+import { Box, Typography, makeStyles } from "@material-ui/core";
+import BeenhereOutlinedIcon from "@material-ui/icons/BeenhereOutlined";
+
+const entryMap = {
+  label: "Project",
+  fields: {
+    project_name: {
+      label: "name",
+    },
+    project_description: {
+      label: "description",
+    },
+    ecapris_subproject_id: {
+      label: "eCAPRIS subproject ID",
+    },
+    current_status: {
+      label: "current status",
+    },
+    current_phase: {
+      label: "phase",
+    },
+    is_deleted: {
+      label: "soft delete status",
+    },
+    milestone_id: {
+      label: "milestone ID",
+    },
+    status_id: {
+      label: "status ID",
+    },
+    // deprecated column, but keeping because historical activities depend on it
+    contractor: {
+      label: "contractor",
+    },
+    project_sponsor: {
+      label: "sponsor ID",
+    },
+    project_website: {
+      label: "project website",
+    },
+    knack_project_id: {
+      label: "knack internal ID",
+    },
+    // deprecated column, but keeping because historical activities depend on it
+    purchase_order_number: {
+      label: "purchase order number",
+    },
+    task_order: {
+      label: "task order",
+    },
+    // deprecated column, but keeping because historical activities depend on it
+    work_assignment_id: {
+      label: "work assignment ID",
+    },
+    parent_project_id: {
+      label: "parent project id",
+    },
+  },
+};
+
+const useStyles = makeStyles((theme) => ({
+  entryText: {
+    padding: "0 0 0 .5rem",
+  },
+  boldText: {
+    fontWeight: 700,
+  },
+  strikeText: {
+    textDecoration: "line-through",
+  },
+}));
+
+const getEntryText = (change, legacyVersion, classes) => {
+  // adding a field
+  if (change.description[0].old === null) {
+    return (
+      <Typography variant="body2" className={classes.entryText}>
+        Added {entryMap.fields[change.description[0].field].label}{" "}
+        {change.description[0].new}
+      </Typography>
+    );
+  }
+
+  // updating a field
+  if (legacyVersion) {
+    return (
+      <Typography variant="body2" className={classes.entryText}>
+        Changed {entryMap.fields[change.description[0].field].label} from "
+        <span className={classes.strikeText}>{change.description[0].old}</span>"{" "}
+        to "{change.description[0].new}"
+      </Typography>
+    );
+  } else {
+    return (
+      <Typography variant="body2" className={classes.entryText}>
+        Changed {entryMap.fields[change.description[0].field].label} from "
+        <span className={classes.strikeText}>
+          {change.description[0].old[change.description[0].field]}
+        </span>
+        " to "{change.description[0].new[change.description[0].field]}";
+      </Typography>
```

Comment:
My inclination is to suggest disregarding this message, because design-by-committee in a PR review is a special kind of uncool, but ... 

![image](https://user-images.githubusercontent.com/3653206/208783640-60f8d8f5-bdf9-4fc5-ae27-fee4243c0357.png) 
👆An example of string-to-a-string type change

Even when the strings are a much more manageable length, the use of a sentence fragment in the `Change` column coupled with the use of strikethrough text decoration on the outgoing value is confusing and at the very least, hard to read. Maybe something like this is clearer? I'm not sure and far from convinced this is best, but it's an idea..

![IMG_2225](https://user-images.githubusercontent.com/3653206/208784910-8723012b-efae-412f-b804-da7662220db9.jpeg)

In terms of maybe an example style to borrow from, the code blocks in this page seem like they could be good textual idea containers. 

http://web.simmons.edu/~grabiner/comm244/weekfour/code-test.html

But,  if a sentence-based description of the change is the way we want to go, I think we should consider forming the sentences just as simply as we can. Like:

The [field name] was changed from "[previous value]" to "[new value]."
ex: The project description was changed from "herd cats" to "runnin' down a dream."